plugin-examples/src/test/java/com/hotels/styx/ExampleMessageInterceptTest.java

```diff
@@ -0,0 +1,34 @@
+package com.hotels.styx;
+
+import com.hotels.styx.api.*;
+import org.testng.annotations.Test;
+import reactor.core.publisher.Mono;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+/**
+ * This tests the behaviours added in the example plugin.
+ */
+
+public class PluginExampleTest {
+    @Test
+    public void returnsEarly() {
+        PluginExample plugin = new PluginExample();
```

Comment:
The class `PluginExample` does not exist. Please ensure that code compiles and tests pass before creating a PR. In the case of these examples, it is important to provide code that we have confirmed functional, so as to avoid confusing anyone who tries to learn from it.